src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -28,6 +28,7 @@
         "symfony/error-handler": "For tracking deprecated interfaces usages at runtime with DebugClassLoader"
     },
     "conflict": {
+        "doctrine/annotations": "<1.10",
```

Comment:
this conflict should also be added in FrameworkBundle, as it is the one configuring the annotation reader service

Reply:
We already have a conflict rule in place there:

https://github.com/symfony/symfony/blob/b23a087d7b4a9b84a1dde314ee9a900dadd741db/src/Symfony/Bundle/FrameworkBundle/composer.json#L73